README.md

```diff
@@ -1 +1,36 @@
-# langchain-assistant
\ No newline at end of file
+# langchain-assistant
+
+## Prerequisites
+
+Make sure you have python and pip installed
```

Comment:
Kindly add the required versions.

What versions should be installed?

Reply:
Updated